src/BehaviorsSDKNative/NativeUnitTests/NativeUnitTests.csproj

```diff
@@ -88,10 +83,12 @@
     <Prefer32Bit>true</Prefer32Bit>
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
+  <PropertyGroup>
+		<OutputPath>..\..\..\out\$(SolutionName)\bin\$(Platform)\$(Configuration)\</OutputPath>
+		<IntermediateOutputPath>..\..\..\out\$(SolutionName)\obj\$(Platform)\$(Configuration)\</IntermediateOutputPath>
+	</PropertyGroup>
   <ItemGroup>
```

Comment:
tabs. And I would move it up to the top property group for consistency

Reply:
I've cleaned up the tabs and moved OutputPath etc to beneath configuration property groups across all the csproj files.